packages/react/src/components/ThemeProvider/GlobalStyle.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { createGlobalCSS } from '@aws-amplify/ui';
+import { Style } from './Style';
+
+interface GlobalStyleProps extends React.StyleHTMLAttributes<{}> {
```

Comment:
Does `style` make sense as the element generic here?

```suggestion
interface GlobalStyleProps extends React.StyleHTMLAttributes<'style'> {
```